src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -58,10 +58,10 @@
  *    `appSearchSource`.
  */
 
-import { setWith } from '@elastic/safer-lodash-set';
```

Comment:
Is there a reason we stopped using the patched `setWith` from `safer-lodash-set`?

Reply:
Oversight! Will fix this